packages/components/src/BlankSlate/BlankSlate.js

```diff
@@ -10,6 +10,17 @@ import { Text } from '../Text';
 import { VStack } from '../VStack';
 import * as styles from './BlankSlate.styles';
 
+/**
```

Comment:
This serves as an example of how we'll type components in the future once we move their types out of the `.d.ts` files.